joomla/joomla-cms
# [4.0] Namespace com_postinstall

Pull Request for Issue # .

### Summary of Changes
This PR namespace com_postinstall component to namespace MVC for Joomla 4.

### Testing Instructions
1. Install Joomla 4 stagging
2. Apply patch
3. Play with Post-installation Messages component and make sure it is working as expected



### Expected result



### Actual result



### Documentation Changes Required
